applications/asset_tracker_v2/src/modules/Kconfig.sensor_module

```diff
@@ -13,7 +13,7 @@ if SENSOR_MODULE
 
 config SENSOR_THREAD_STACK_SIZE
 	int "Sensor module thread stack size"
-	default 1152
+	default 4096
```

Comment:
I think we should only increase the stack whic much if it's strictly needed, so this can probably be 
```
default 4096 if BSEC
default 1152
```
Though I'm not sure if this large stack is required both for the sensor module thread and the BSEC thread?

Reply:
I forgot to run the thread analyzer on this. Will update.